src/lib/unlang/try_priv.h

```diff
@@ -19,7 +19,7 @@
  * $Id$
  *
  * @file unlang/try_priv.h
- * @brief Declarations for unlang trys
+ * @brief Declarations for unlang tries
```

Comment:
Incorrect.  This is for try blocks.  Could be changed to `try keywords`

Reply:
Or perhaps:
```suggestion
 * @brief Declaration for unlang try
```